tox.ini

```diff
@@ -1,41 +0,0 @@
-[tox]
-envlist = py311
-skip_missing_interpreters = True
-skipsdist = True
-
-[testenv]
-sitepackages = True
-deps =
-    -r{toxinidir}/requirements.txt
-    -r{toxinidir}/test_requirements.txt
-commands =
-    py.test {posargs}
-    {toxinidir}/scripts/test/detect_missing_migrations.sh
-    {toxinidir}/scripts/test/no_auto_migrations.sh
-
-passenv =
-    DATABASE_URL
-    MEDIA_ROOT
-    CELERY_BROKER_URL
-    CELERY_RESULT_BACKEND
-    OPEN_DISCUSSIONS_REDDIT*
-    PORT
-    TRAVIS*
-    CI
-setenv =
-    DEBUG=False
-    CELERY_TASK_ALWAYS_EAGER=True
-    SENTRY_DSN=
-    OPEN_DISCUSSIONS_COOKIE_NAME=cookie_monster
-    OPEN_DISCUSSIONS_COOKIE_DOMAIN=localhost
-    OPEN_DISCUSSIONS_DB_DISABLE_SSL=True
-    OPEN_DISCUSSIONS_DEFAULT_SITE_KEY=mm_test
-    OPEN_DISCUSSIONS_SECURE_SSL_REDIRECT=False
-    OPEN_DISCUSSIONS_NOTIFICATION_EMAIL_BACKEND=anymail.backends.test.EmailBackend
-    OPEN_DISCUSSIONS_BASE_URL=http://localhost:8063/
-    MAILGUN_SENDER_DOMAIN=other.fake.site
-    MAILGUN_KEY=fake_mailgun_key
-    OPENSEARCH_URL=http://localhost:9300/
-    OPENSEARCH_INDEX=testindex
-    INDEXING_API_USERNAME=mitodl
-    OCW_WEBHOOK_KEY=fake_key
```

Comment:
Some of these aren't in the ci.yml file and/or pytest.ini file, might be good to make them all consistent?  For example `OPEN_DISCUSSIONS_NOTIFICATION_EMAIL_BACKEND` (xpro, mitxonline have the equivalent env in both ci.yml and pytest.ini)